### PR TITLE
Tracking of JWT Token Expiration

### PR DESCRIPTION
**Pull request by herzykj**
- ability to rotate JWT token for sandbox
- display JWT expiration in multiple locations 
- changes in wording displays in console output

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,222 +90,32 @@ A tool that makes it easier to manage Akamai Sandboxes. Call the Sandbox API fro
...
 
 Commands:
   help [command]                                                Displays help information for the given command.
   install                                                       Downloads and installs the Sandbox Client software.
-  list|ls [options]                                             Lists sandboxes that you have managed locally.
-  show [sandbox-identifier]                                     Provides details about a sandbox.
-  rules [sandbox-identifier]                                    Shows rule tree for sandbox.
-  use <sandbox-identifier>                                      Sets the identified sandbox as currently active.
+  list|ls [options]                                             Lists sandboxes that are available locally.
+  show [sandbox-id]                                             Shows details about the sandbox and JWT expiration date.
```
Comment (krzyk) [inline README.md:102]: why `the sandbox` here and `a sandbox` in other places?
there are few other places that use `the sandbox`

File: README.md [hunk 2]
```diff
@@ -91,222 +90,32 @@ A tool that makes it easier to manage Akamai Sandboxes. Call the Sandbox API fro
...
   --section <name>                                              Use this section in edgerc file that contains the credential set.
   --accountkey <account-id>                                     Use given internal parameter.
   -h, --help                                                    Output usage information.
 
 Commands:
   help [command]                                                Displays help information for the given command.
   install                                                       Downloads and installs the Sandbox Client software.
-  list|ls [options]                                             Lists sandboxes that you have managed locally.
-  show [sandbox-identifier]                                     Provides details about a sandbox.
-  rules [sandbox-identifier]                                    Shows rule tree for sandbox.
```
Comment (krzyk) [inline README.md:104]: `for sandbox` shouldn't it be `for a sandbox`?

File: README.md [hunk 3]
```diff
@@ -91,222 +90,32 @@ A tool that makes it easier to manage Akamai Sandboxes. Call the Sandbox API fro
...
-  use <sandbox-identifier>                                      Sets the identified sandbox as currently active.
+  list|ls [options]                                             Lists sandboxes that are available locally.
+  show [sandbox-id]                                             Shows details about the sandbox and JWT expiration date.
+  rules [sandbox-id]                                            Shows a rules tree for the sandbox.
```
Comment (krzyk) [inline README.md:103]: `a rules` -> `rules`

File: src/utils/cli-utils.ts [hunk 4]
```diff
@@ -60,3 +60,13 @@ export async function spinner(func, userMsg: string = '') {
...
 }
+
+export function dateToString(date){
+  const options = {
+    year: 'numeric', month: 'short', day: 'numeric',
+    hour: 'numeric', minute: 'numeric',
+    timeZone: 'UTC',
+    timeZoneName: 'short'
+  };
+  return new Intl.DateTimeFormat('en-US', options).format(date)
```
Comment (krzyk) [inline src/utils/cli-utils.ts:71]: just an idea, as I didn't trace where it is used, but I think we should use user locale to determine the date time format not hardcode it to en-US (unless it is for sending data to the backend).

**Review by krzyk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (herzykj) [reply README.md:102]: a sandbox is used when you are not refering to concrete sandbox instance (creates a new sandbox)  
The sandbox is used when it describes concrete instance defined by ID

File: README.md (hunk 3 above)
Comment (herzykj) [reply README.md:103]: rules tree is a singular. 
so Shows a rules tree   OR Shows rules treeS

**Review by krzyk — APPROVED** [review]

**Pull request merged**